app/javascript/components/common/PartyElement.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+import Avatar from 'antd/es/avatar';
+import Button from 'antd/es/button';
+
+export default function PartyElement({ party, theme }) {
```

Comment:
Also, please double check whether we have new information regarding each party in figma

Reply:
AvatarList component is already responsible for listing the parties. Given that, I think this component is no longer necessary?

Reply:
Agree @nancruz , I'm gonna apply some of the changes on it. Plus, AvatarList is a more correct naming since this component will also include party members